syntax = "proto3";
package disc;

service DiscoveryService {
  rpc Hello(HelloRequest) returns (HelloReply);
}

message HelloRequest {
  int32 sender_id = 1;
}

message HelloReply {
  int32 node_id = 1;
  double energy = 2;
}
